src/BottomNavigation/BottomNavigation.js

```diff
@@ -0,0 +1,54 @@
+// @flow weak
+
+import React, { PropTypes, Children, cloneElement } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('BottomNavigation', (theme) => {
+  return {
+    root: {
+      display: 'flex',
+      justifyContent: 'center',
+      height: 56,
+      backgroundColor: theme.palette.background.paper,
+    },
+  };
+}, { index: 5 });
```

Comment:
@pradel I've changed this once again so it's much easier to keep track of. It's all set in one place now in `MuiThemeProvider`.